modules/install-acs-operator.adoc

```diff
@@ -30,6 +30,12 @@ Using the OperatorHub provided with {ocp} is the easiest way to install {product
 If you choose automatic updates, when a new version of the Operator is available, Operator Lifecycle Manager (OLM) automatically upgrades the running instance of your Operator.
 +
 If you choose manual updates, when a newer version of the Operator is available, OLM creates an update request. As a cluster administrator, you must then manually approve that update request to update the Operator to the new version.
++
+[IMPORTANT]
+====
+If you choose manual updates, you should update the {product-title-short} Operator in all
+secured clusters when you update the {product-title-short} Operator in the cluster where Central is installed. The secured clusters and the cluster where Central is installed should have the same version to ensure optimal
```

Comment:
```suggestion
secured clusters when you update the {product-title-short} Operator in the cluster where Central is installed. The secured clusters and the cluster where Central is installed should have the same version to ensure optimal functionality.
```

Reply:
Whoops, must've cut that off on accident. It's fixed now, thanks 👍